pdns/pdns_recursor.cc

```diff
@@ -2412,11 +2412,13 @@ static void startDoResolve(void *p)
       g_log<<endl;
     }
 
-    if (sr.d_outqueries || sr.d_authzonequeries) {
-      g_recCache->cacheMisses++;
-    }
-    else {
-      g_recCache->cacheHits++;
+    if (dc->d_mdp.d_header.opcode != Opcode::Notify) {
```

Comment:
I _think_ this makes more sense, we reject other OPCODEs earlier.

```suggestion
    if (dc->d_mdp.d_header.opcode == Opcode::Query) {
```

Reply:
It's equivalent, but it might read better indeed.